src/WireMock.Net/Matchers/JmesPathMatcher.cs

```diff
@@ -74,7 +74,7 @@ public MatchResult IsMatch(string? input)
         var score = MatchScores.Mismatch;
         Exception? exception = null;
 
-        if (input != null)
+        if (!string.IsNullOrWhiteSpace(input))

```

Comment:
Same for this.

See
https://github.com/WireMock-Net/WireMock.Net/blob/master/test/WireMock.Net.Tests/Matchers/JmesPathMatcherTests.cs#L55